app/brave_main_delegate.cc

```diff
@@ -139,17 +140,15 @@ bool BraveMainDelegate::BasicStartupComplete(int* exit_code) {
       extensions_features::kNewExtensionUpdaterService.name,
 #endif
       features::kDesktopPWAWindowing.name,
+      omnibox::kSimplifyHttpsIndicator.name,
   };
 
   // Disabled features.
   const std::unordered_set<const char*> disabled_features = {
       autofill::features::kAutofillSaveCardSignInAfterLocalSave.name,
       autofill::features::kAutofillServerCommunication.name,
-      features::kAudioServiceOutOfProcess.name,
```

Comment:
Removing since this feature is disabled by default. https://cs.chromium.org/chromium/src/content/public/common/content_features.cc?l=51

The test in to check if this feature is disabled is still available in `brave_main_delegate_browsertest.cc` to alert if this feature gets enabled in future. cc: @darkdh @mkarolin